planner/core/logical_plan_builder.go

```diff
@@ -1768,7 +1768,8 @@ func (b *PlanBuilder) checkOnlyFullGroupByWithGroupClause(p LogicalPlan, sel *as
 		}
 		switch errExprLoc.Loc {
 		case ErrExprInSelect:
-			return ErrFieldNotInGroupBy.GenWithStackByArgs(errExprLoc.Offset+1, errExprLoc.Loc, sel.Fields.Fields[errExprLoc.Offset].Text())
+			errText := fmt.Sprintf("%s.%s.%s", col.DBName.O, col.OrigTblName.O, col.OrigColName.O)
```

Comment:
Please remove the `fmt.Sprint`. It's inefficient. Just plus the string and no need to declare a new variable either.

Reply:
fixed